supplier/tests/test_supplier.py

```diff
@@ -28,7 +28,7 @@ def test_delete_exist_supplier(self, supplier0):
     @pytest.mark.django_db()
     def test_fail_to_save_supplier(self, supplier0):
         with pytest.raises(ValidationError):
-            supplier0.user_name = ''
+            supplier0.user_name = '123'
```

Comment:
Why do we change this?

Reply:
It is the same thibg but we want to get validationerror of something less than 6 so i wrote that,also username has to be set to pass the account lines other than that valueerror is raised

Reply:
ok